deploy/examples/se_demo/main.tf

```diff
@@ -127,6 +127,13 @@ module "db_onboarding" {
   ]
 }
 
+module "statistics" {
+  source = "../../modules/statistics"
+  depends_on = [
+    module.gw_attachments
```

Comment:
Just want to verify that we will only get metrics for those who succeeded to run "gw federations"

Reply:
I set to run stats module at the beginning of the deployment at both recipes